frontend/package.json

```
@@ -38,6 +38,7 @@
 		"@fortawesome/fontawesome-svg-core": "1.2.10",
 		"@fortawesome/free-solid-svg-icons": "5.6.1",
 		"@fortawesome/react-fontawesome": "0.1.3",
+		"@hookform/resolvers": "^2.8.5",
```

We are using fixed  versions
```suggestion
		"@hookform/resolvers": "2.8.5",
```